run-uts

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Please put this in the Makefile - I don't think it's good to have a whole extra file just for three lines